ui/perfherder/graphs/GraphsView.jsx

```diff
@@ -94,7 +93,7 @@ class GraphsView extends React.Component {
       highlightCommonAlerts,
       highlightChangelogData,
       highlightedRevisions,
-    } = queryString.parse(this.props.location.search);
+    } = new URLSearchParams(this.props.location.search).getAll();
```

Comment:
I get a console error for this line. `getAll()` seems to require an argument.
![image](https://user-images.githubusercontent.com/63001299/210561923-ce6a813e-4bd7-44bd-ba76-d6942ed25903.png)
![image](https://user-images.githubusercontent.com/63001299/210562230-58bbbd9b-f263-4233-8369-0298410d5ebc.png)